### PR TITLE
VZ-4522 Add troubleshooting page for Istio Ingress Gateway

### PR DESCRIPTION
**Pull request by davisgomes**
Add troubleshooting redirect for the VZ analyze tool. This is in tandem with an analysis change to detect when the Istio Ingress Gateway Service is missing an external IP

### CODE REVIEW

**Review by rosemarymarano — APPROVED** [review]
LGTM

File: content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioIngressNoLoadBalancerIP.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Istio Ingress Controller No Load Balancer IP
+linkTitle: Istio Ingress Controller No Load Balancer IP
+description: Analysis detected Istio load balancer missing ingress IP address
```
Comment (mbenmclean) [inline content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioIngressNoLoadBalancerIP.md:4]: I might suggest
`Analysis detected Istio Ingress Gateway is missing a Load Balancer IP address`

File: content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioIngressNoLoadBalancerIP.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: Istio Ingress Controller No Load Balancer IP
```
Comment (mbenmclean) [inline content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioIngressNoLoadBalancerIP.md:2]: Istio Ingress Gateway No Load Balancer IP

File: content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioIngressNoLoadBalancerIP.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+linkTitle: Istio Ingress Controller No Load Balancer IP
```
Comment (mbenmclean) [inline content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioIngressNoLoadBalancerIP.md:3]: Istio Ingress Gateway No Load Balancer IP

File: content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioIngressNoLoadBalancerIP.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+description: Analysis detected Istio load balancer missing ingress IP address
+weight: 5
+draft: false
+---
+
+### Summary
+Analysis detected that the Verrazzano installation failed while installing the Istio Ingress Gateway.
+
+The root cause appears to be that the load balancer is either missing or unable to set the ingress IP address on the Istio Ingress Gateway service.
```
Comment (mbenmclean) [inline content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioIngressNoLoadBalancerIP.md:12]: unable to set the public IP address

Comment (davisgomes) [reply content/en/docs/troubleshooting/diagnostictools/analysisadvice/IstioIngressNoLoadBalancerIP.md]: I updated all of these phrases with the modification of "external IP" which I think is more accurate to Kubernetes terms.

**Review by mbenmclean — APPROVED** [review]

**Review by MelwynPaul — APPROVED** [review]
Looks good.

**Pull request merged**